app/src/main/res/values-en/strings.xml

```diff
@@ -0,0 +1,40 @@
+<resources>
+    <string name="main_activity_description">Chat with your mentors.</string>
```

Comment:
I'd delete the period here, it's in the source but depending on where it is in the app, it may look weird next to "Chat with a mentor" without any punctuation.